src/Compilers/CSharp/Portable/Compiler/MethodCompiler.cs

```diff
@@ -1742,13 +1742,15 @@ public override object VisitField(FieldSymbol symbol, TypeCompilationState argum
 
             BoundBlock? body;
             NullableWalker.VariableState? nullableInitialState = null;
+            BoundStatement? constructorInitializer;
 
             initializersBody ??= GetSynthesizedEmptyBody(method);
 
             if (method is SynthesizedRecordConstructor recordStructPrimaryCtor && method.ContainingType.IsRecordStruct)
             {
                 body = BoundBlock.SynthesizedNoLocals(recordStructPrimaryCtor.GetSyntax());
                 nullableInitialState = getInitializerState(body);
+                constructorInitializer = BindImplicitConstructorInitializerIfAny(method, compilationState, diagnostics);
```

Comment:
>BindImplicitConstructorInitializerIfAny(method, compilationState, diagnostics); [](http://example.com/codeflow?start=41&length=79)

Is this ever going to produce anything, but null? It looks like struct constructors do not have implicit initializers